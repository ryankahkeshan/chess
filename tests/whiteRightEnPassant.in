setup
+ K e1
+ k e8
+ P d4
+ p c7
+ p e7
done
game human human
move d4 d5
move e8 d7

move d5 c6
move d5 e6

move e1 e2
move c7 c6
move e2 e1
move c6 c5

move d5 c6
move e1 e2
move e7 e5
move d5 e6
