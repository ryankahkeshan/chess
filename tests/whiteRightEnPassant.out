8  _ _ _ _
7 _ _ _ _ 
6  _ _ _ _
5 _ _ _ _ 
4  _ _ _ _
3 _ _ _ _ 
2  _ _ _ _
1 _ _ K _ 

  abcdefgh

8  _ _k_ _
7 _ _ _ _ 
6  _ _ _ _
5 _ _ _ _ 
4  _ _ _ _
3 _ _ _ _ 
2  _ _ _ _
1 _ _ K _ 

  abcdefgh

8  _ _k_ _
7 _ _ _ _ 
6  _ _ _ _
5 _ _ _ _ 
4  _ P _ _
3 _ _ _ _ 
2  _ _ _ _
1 _ _ K _ 

  abcdefgh

8  _ _k_ _
7 _ p _ _ 
6  _ _ _ _
5 _ _ _ _ 
4  _ P _ _
3 _ _ _ _ 
2  _ _ _ _
1 _ _ K _ 

  abcdefgh

8  _ _k_ _
7 _ p p _ 
6  _ _ _ _
5 _ _ _ _ 
4  _ P _ _
3 _ _ _ _ 
2  _ _ _ _
1 _ _ K _ 

  abcdefgh

8  _ _k_ _
7 _ p p _ 
6  _ _ _ _
5 _ _ _ _ 
4  _ P _ _
3 _ _ _ _ 
2  _ _ _ _
1 _ _ K _ 

  abcdefgh

8  _ _k_ _
7 _ p p _ 
6  _ _ _ _
5 _ _P_ _ 
4  _ _ _ _
3 _ _ _ _ 
2  _ _ _ _
1 _ _ K _ 

  abcdefgh

8  _ _ _ _
7 _ pkp _ 
6  _ _ _ _
5 _ _P_ _ 
4  _ _ _ _
3 _ _ _ _ 
2  _ _ _ _
1 _ _ K _ 

  abcdefgh

Invalid move
Invalid move
8  _ _ _ _
7 _ pkp _ 
6  _ _ _ _
5 _ _P_ _ 
4  _ _ _ _
3 _ _ _ _ 
2  _ _K_ _
1 _ _ _ _ 

  abcdefgh

8  _ _ _ _
7 _ _kp _ 
6  _p_ _ _
5 _ _P_ _ 
4  _ _ _ _
3 _ _ _ _ 
2  _ _K_ _
1 _ _ _ _ 

  abcdefgh

8  _ _ _ _
7 _ _kp _ 
6  _p_ _ _
5 _ _P_ _ 
4  _ _ _ _
3 _ _ _ _ 
2  _ _ _ _
1 _ _ K _ 

  abcdefgh

8  _ _ _ _
7 _ _kp _ 
6  _ _ _ _
5 _ pP_ _ 
4  _ _ _ _
3 _ _ _ _ 
2  _ _ _ _
1 _ _ K _ 

  abcdefgh

Invalid move
8  _ _ _ _
7 _ _kp _ 
6  _ _ _ _
5 _ pP_ _ 
4  _ _ _ _
3 _ _ _ _ 
2  _ _K_ _
1 _ _ _ _ 

  abcdefgh

8  _ _ _ _
7 _ _k_ _ 
6  _ _ _ _
5 _ pPp _ 
4  _ _ _ _
3 _ _ _ _ 
2  _ _K_ _
1 _ _ _ _ 

  abcdefgh

8  _ _ _ _
7 _ _k_ _ 
6  _ _P_ _
5 _ p _ _ 
4  _ _ _ _
3 _ _ _ _ 
2  _ _K_ _
1 _ _ _ _ 

  abcdefgh

Black is in check.
Final Score:
White: 0
Black: 0
